atom/browser/api/atom_api_web_contents_impl.cc

```diff
@@ -16,6 +19,13 @@ namespace atom {
 
 namespace api {
 
+void WebContents::DetachFromOuterFrame() {
+  // See detach_webview_frame.patch on how to detach.
+  auto* impl = static_cast<content::WebContentsImpl*>(web_contents());
+  impl->GetRenderManagerForTesting()->RemoveOuterDelegateFrame();
```

Comment:
Probably a better way to access the frame host manager would be via `FrameTreeNode` https://cs.chromium.org/chromium/src/content/browser/frame_host/frame_tree_node.h?l=106 . Based on the comments `GetRenderManagerForTesting` might be removed soon.